React-native.md

```diff
@@ -0,0 +1,84 @@
+# React-Native 
+
+React Native is a JavaScript framework Allow you to writes real, natively rendering mobile applications for iOS and Android ,based on React.
+You can create code just once and use it to power both their iOS and Android apps. This translates to huge time and resource savings. ,React Native has the ability to render APIs in Objective-C (for iOS) or Java (for Android),your application will render using “bridge” to become real mobile UI components,and will look and feel like any other mobile application,your React Native apps can access platform features like the phone camera, or the user’s location.
+
+![Test Image 1](https://www.netguru.com/hs-fs/hubfs/image10-Jul-03-2020-11-20-54-82-AM.png?width=1600&name=image10-Jul-03-2020-11-20-54-82-AM.png)
+
+Now we can start building our first React Native App ,to do that you can follow the steps below:
```

Comment:
This is excellent! It is always good to start writing code as soon as possible in a workshop 👍